src/Client.cpp

```diff
@@ -183,6 +210,11 @@ void Client::OnAfterCreated(CefRefPtr<CefBrowser> browser)
 
     CEF_REQUIRE_UI_THREAD();
 
+    if (!m_requestHandler->m_messageRouterBrowserSide) {
```

Comment:
Would it still need to be inside an if statement in `CefClient::OnContextInitialized`?